apps/graph/list/color_list.cpp

```diff
@@ -0,0 +1,44 @@
+#include "color_list.h"
+#include "apps/i18n.h"
+
+namespace Graph {
+    namespace FunctionColors {
```

Comment:
Why use a new namespace ?

Reply:
coding style :rofl: it was more clear for me but i can remove it if you need